content/blog/insights-resources-v2/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Updated Resources View"
```

Comment:
title seems a little bland. Maybe something like:

`introducing Resources view V2`

Reply:
Changed to 'Introducing the new Resources view'